signals/generators/ios/template_methods.py

```diff
@@ -66,10 +66,10 @@ def key_path(api):
 
 
 def get_object_name(request_object, upper_camel_case=False):
```

Comment:
If we're planning to use this for anything else, it might make more sense to have a `to_camel_case` function that just takes a string and doesn't assume an object with attribute `name`. I just left it this way because it was the closest translation of what we already had, but can update if you think that's a good idea.